pkgs/development/python-modules/numpy/default.nix

```diff
@@ -5,23 +5,22 @@
 , gfortran
 , pytest
 , blas
+, lapack
 , writeTextFile
 , isPyPy
 , cython
 , setuptoolsBuildHook
  }:
 
+assert (!blas.is64bit) && (!lapack.is64bit);
+
 let
-  blasImplementation = lib.nameFromURL blas.name "-";
   cfg = writeTextFile {
     name = "site.cfg";
     text = (lib.generators.toINI {} {
-      ${blasImplementation} = {
-        include_dirs = "${blas}/include";
-        library_dirs = "${blas}/lib";
-      } // lib.optionalAttrs (blasImplementation == "mkl") {
-        mkl_libs = "mkl_rt";
-        lapack_libs = "";
+      ${blas.implementation} = {
+        include_dirs = "${blas}/include:${lapack}/include";
+        library_dirs = "${blas}/lib:${lapack}/lib";
       };
```

Comment:
Just checking that the missing `mkl_libs = "mkl_rt";` is intentional?

Reply:
As far as I can tell that is indeed still needed.